.github/workflows/build.yml

```diff
@@ -133,11 +129,10 @@ jobs:
           # https://github.com/facebook/react-native/issues/30036 and
           # https://github.com/microsoft/react-native-macos/issues/620 for more
           # details.
-          yarn set-react-version main
+          npm run set-react-version main
```

Comment:
interesting, why is this going back to npm? Some yarn3 change that makes it more consistent to just go back to npm run?

Reply:
Yarn 3 doesn't allow you to `run` unless you've done a `yarn install` first. Something about security. I couldn't find an opt-out, so `npm run` it is 😄